halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Features.java

```diff
@@ -22,7 +22,7 @@
 
 @Data
 @EqualsAndHashCode(callSuper = false)
-@JsonIgnoreProperties({"jobs"})
+@JsonIgnoreProperties({"jobs", "managedPipelineTemplatesV2UI"})
```

Comment:
Just followed the link from the Deck PR specifically to make sure this was here but you are waaay ahead of me
:heart: